content/reports/1000_IBM Power - Virtual Machines/010_All Power VMs.yaml

```diff
@@ -0,0 +1,86 @@
+---
```

Comment:
Looks like this files and `content/reports/1000_IBM Power - Virtual Machines/011_All PowerHMC VMs.yaml` are essentially duplicates (only diff is the `menu_name:` values). I think we would want to keep only the file titled "All PowerHMC VMs"